packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+```
+
+#### Response example
+
+```json
+{
+  "id": "irb1qe6PGuMc7Oh8N0g4",
+  "_links": {
+    "self": {
+      "href": "https://${yourOktaDomain}/api/v1/users/00uuk41Hjga5qGfQ30g3"
+    }
+  }
+}
+```
+
+### Delete a member from a binding
+<ApiLifecycle access="beta" />
+
+<ApiOperation method="delete" url="/api/v1/iam/resource-sets/${resourceSetId}/bindings/${roleId}/members/${memberId}" />
+
+Delete a Member of a Role in a Resource Set
+
+#### Request parameters
+
+| Parameter      | Description                                                           | Param Type   | DataType       | Required |
+| :------------- | :-------------------------------------------------------------------- | :----------- | :------------- | :------- |
+| resourceSetId  | `id` of the target Resource Set                                       | URL          | String         | TRUE     |
+| roleId         | `id` of the Role to identify the Binding                              | URL          | String         | TRUE     |
+| memberId       | `id` of the Member within the Binding                                 | URL          | String         | TRUE     |
+
+
+`memberId` is the id obtained when [listing Members within Binding](#list-members-in-a-binding). For example, if the Member object was:
+```json
+{
+  "id": "irb1qe6PGuMc7Oh8N0g4",
+  "_links": {
+    "self": {
+      "href": "https://${yourOktaDomain}/api/v1/users/00uuk41Hjga5qGfQ30g3"
+    }
+  }
+}
+```
+`irb1qe6PGuMc7Oh8N0g4` could be used as `memberId` to remove the User from list of Members in the Binding.
+
+#### Response parameters
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+#### Request example
+
+```bash
+curl -v -X DELETE \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/bindings/cr0WxyzJxGIr0ouum0g4/members/irb1qe6PGuMc7Oh8N0g4"
+```
+
+#### Response example
+
+``` http
+HTTP/1.1 204 No Content
+```
+
+### Retrieve bindings
+<ApiLifecycle access="beta" />
+
+#### Get a binding by role id
+<ApiLifecycle access="beta" />
+
+<ApiOperation method="get" url="/api/v1/iam/resource-sets/${resourceSetId}/bindings/${roleId}" />
+
+Get a Binding from a Resource Set by its Role id
+
+##### Request parameters
+
+| Parameter      | Description              | Param Type   | DataType     | Required |
+| :------------- | :----------------------- | :----------- | :----------- | :------- |
+| resourceSetId  | `id` of the Resource Set | URL          | String       | TRUE     |
+| roleId         | `id` of the Role         | URL          | String       | TRUE     |
+
+##### Response parameters
+
+The `id` of the Role as well as the following `_links`:
+* `self` gets this Role's Binding within the Resource Set
+* `bindings` get a paginable list of Role Bindings in the Resource Set
+* `resource-set` gets Resource Set
```

Comment:
Resource Set > the Resource Set

Add a period at the end of each bullet item here as they're all sentences.